src/main/host/syscall/format.rs

```diff
@@ -525,12 +526,17 @@ where
                 let rv = SyscallVal::<'_, RV>::from_mem(*x, self.options, self.mem);
                 write!(f, "{}", rv)
             }
-            SyscallResult::Err(SyscallError::Errno(e)) => {
+            SyscallResult::Err(SyscallError::Failed(failed)) => {
+                let errno: Errno = failed.errno;
                 let rv = SysCallReg {
-                    as_i64: -(*e as i64),
+                    as_i64: -(errno as i64),
                 };
                 let rv = SyscallVal::<'_, RV>::from_mem(rv, self.options, self.mem);
-                write!(f, "{} ({})", rv, e)
+                if errno == Errno::EINTR {
+                    write!(f, "{} ({}, restartable:{})", rv, errno, failed.restartable)
+                } else {
+                    write!(f, "{} ({})", rv, errno)
+                }
```

Comment:
I think for the strace syscall formatting we don't need to print the `restartable` flag.

Reply:
Yeah, agreed. Done.